WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -450,4 +463,25 @@ protected void onPostExecute(Boolean result) {
         }
     }
 
-}
+    public HashSet<Long> getSelectedCommentsId() {
+        return mSelectedCommentsId;
+    }
+
+
+    public CommentAdapterState getAdapterState() {
+        return new CommentAdapterState(mSelectedCommentsId, mModeratingCommentsIds);
+    }
+
+    public void setInitialState(CommentAdapterState adapterState) {
+        if (adapterState == null) return;
+
+        if (adapterState.hasSelectedComments()) {
```

Comment:
Do `mSelectedCommentsId` and `mModeratingCommentsIds` need to be cleared here before adding to them?